Lib/platform.py

```diff
@@ -170,10 +172,13 @@ def libc_ver(executable=sys.executable, lib='', version='',
                 m = _libc_search.search(binary, pos)
             else:
                 m = None
-            if not m:
-                binary = f.read(chunksize)
-                if not binary:
-                    break
+            if not m or (not eof and m.end() == len(binary)):
+                chunk = f.read(chunksize)
+                if not chunk:
+                    if not m:
+                        break
+                    eof = True
+                binary = binary[max(pos, len(binary) - 1000):] + chunk
```

Comment:
IIUC, the added logic is to fix the bug the version info might just appear around the boundary of chunks. But according to the documentation and comment, it seems the original intention is to let the user choose an appropriate chunk size. If we change the logic, the parameter `chunksize` looks somewhat unnecessary.
BTW, the default argument of `chunksize` conflicts with implementation. (2048 vs 16384)

Reply:
What are the documentation and comment about the original intention? I can't find anything.

Reply:
I just infer from it. Otherwise I don't understand what's `chunksize` for. Anyway, it doesn't matter. I just think after this change, exposing `chunksize` seems somewhat superfluous.